README.md

```diff
@@ -14,7 +14,11 @@
 1. Vytvořit novou MySQL databázi a importovat `events.sql.zip`
 2. Zkopírovat `app/config/templates/config.local.neon` do adresáře `app/config` a upravit konfiguraci
 3. Zkopírovat `phinx.yml.template` jako nový soubor `phinx.yml` a nastavit přístupy do databáze (viz [Phinx docs](http://docs.phinx.org/en/latest/configuration.html))
-4. Nainstalovat závislosti `composer install` a `bower install`
+4. Nainstalovat závislosti
+    ```
+    composer install
+    bower install
+    ```
```

Comment:
I didn't notice this, so I've split it to 2 lines that are easy to copy, paster and run.